lensless/utils/dataset.py

```diff
@@ -1599,22 +1625,45 @@ def _get_images_pair(self, idx):
                 shape=(self.alignment["height"], self.alignment["width"], 3),
                 interpolation=cv2.INTER_NEAREST,
             )
+            background = (
+                resize(
+                    background_np,
+                    shape=(self.alignment["height"], self.alignment["width"], 3),
+                    interpolation=cv2.INTER_NEAREST,
+                )
+                if not None
+                else None
+            )
         elif self.display_res is not None:
             lensed = resize(
                 lensed_np, shape=(*self.display_res, 3), interpolation=cv2.INTER_NEAREST
             )
+            background = (
```

Comment:
remove, it's a special case for preparing the lensless data **not** the lensless data